src/web/components/App.tsx

```diff
@@ -327,7 +336,49 @@ export const App = ({ CAPI, NAV, ophanRecord }: Props) => {
 				pubData,
 			});
 		}
-	}, [countryCode, CAPI.config.switches.consentManagement, pageViewId]);
+	}, [
+		countryCode,
+		CAPI.config.switches.consentManagement,
+		pageViewId,
+		browserId,
+	]);
+
+	// ************************
+	// *      Commercial      *
+	// ************************
+	const [sentCommercialMetrics, setSentCommercialMetrics] = useState<boolean>(
+		false,
+	);
```

Comment:
You might find `useOnce` works as an alternative here, rather than having state to ensure something only occurs once.

Reply:
https://github.com/guardian/dotcom-rendering/blob/44bf079a459f5c29a5c3a03d946bbd28dd5df65a/src/web/lib/useOnce.ts